components/registry-facade/cmd/run.go

```diff
@@ -189,6 +190,26 @@ var runCmd = &cobra.Command{
 	},
 }
 
+func loadDockerCfg(fn string) *configfile.ConfigFile {
+	if tproot := os.Getenv("TELEPRESENCE_ROOT"); tproot != "" {
+		fn = filepath.Join(tproot, fn)
+	}
+	fr, err := os.OpenFile(fn, os.O_RDONLY, 0)
+	if err != nil {
+		log.WithError(err).Fatal("cannot read docker auth config")
```

Comment:
Should we return here?

Reply:
We could to make the code clearer. Because the log is `Fatal` we'll end the process at this point.

Reply:
Ops, you are right! I missed it.